packages/assets-controllers/CHANGELOG.md

```diff
@@ -6,6 +6,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [24.0.0]
+### Added
+- Add `getDefaultTokenListState` function to `TokenListController` ([#3744](https://github.com/MetaMask/core/pull/3744))
+- Add `getDefaultNftState` function to the `NftController` ([#3742](https://github.com/MetaMask/core/pull/3742))
+- Add `getDefaultTokensState` function to the `TokensController` ([#3743](https://github.com/MetaMask/core/pull/3743))
+
+### Changed
+- Price API perf improvements ([#3753](https://github.com/MetaMask/core/pull/3753), [#3755](https://github.com/MetaMask/core/pull/3755))
+  - Reduce token batch size from 100 to 30
+  - Sort token addresses in query params for more cache hits
+- **BREAKING:** Bump `@metamask/preferences-controller` to ^6.0.0
```

Comment:
Nit: We usually list breaking changes first, as sometimes they are the only changes that the reader cares about:
```suggestion
- **BREAKING:** Bump `@metamask/preferences-controller` to ^6.0.0
- Price API perf improvements ([#3753](https://github.com/MetaMask/core/pull/3753), [#3755](https://github.com/MetaMask/core/pull/3755))
  - Reduce token batch size from 100 to 30
  - Sort token addresses in query params for more cache hits
```